packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/RecentSubjectsContainer.js

```diff
@@ -0,0 +1,76 @@
+import asyncStates from '@zooniverse/async-states'
+import counterpart from 'counterpart'
+import { inject, observer } from 'mobx-react'
+import { bool } from 'prop-types'
+import React, { Component } from 'react'
+
+import RecentSubjects from './RecentSubjects'
+import MessageBox from './components/MessageBox'
+import Placeholder from './components/Placeholder'
+import fetchRecentSubjects from './helpers/fetchRecentSubjects'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function storeMapper (stores) {
+  return {
+    projectId: stores.store.project.id
+  }
+}
+
+@inject(storeMapper)
+@observer
+class RecentSubjectsContainer extends Component {
+  constructor () {
+    super()
+    this.state = {
+      loading: asyncStates.initialized,
+      subjects: []
+    }
+  }
+
+  componentDidMount () {
+    this.setState({ loading: asyncStates.loading })
+    fetchRecentSubjects(this.props.projectId)
```

Comment:
If the subjects are hidden on small screen sizes, then we shouldn't be making an API request for them. Could you check for this before making the request?

Reply:
We don't hide subjects on phones, we just show them at a smaller size. I've changed the code to reflect how the site works at the moment.

Reply:
Check in with @beckyrother, but I don't think she wants to match the current site, but wants to hide them on mobile instead now with the new design?

Reply:
Ok. There's nothing in this PR to indicate how the widget should look, so I assumed it would be the same as the current site.